Quantum-Control-Applications/ESR/input_calibration.py

```diff
@@ -0,0 +1,75 @@
+from qm import SimulationConfig
```

Comment:
I think this file is the same as 'raw_adc_traces.py' in the SC folder maybe we can rename it for consistency

Reply:
the problem is that it does not measure anything meaningful just the offsets